File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/MoveMessage.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;

import java.io.Serializable;

public class MoveMessage implements Serializable, Message {
    private AuthorizationCredentials authorization;
    private Move move;

    public AuthorizationCredentials getAuthorization() {
        return authorization;
    }

    public void setAuthorization(AuthorizationCredentials authorization) {
        this.authorization = authorization;
    }

    public Move getMove() {
        return move;
    }

    public void setMove(Move move) {
        this.move = move;
    }
}

File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/MoveMessage.js

class MoveMessage {
    constructor(authorization, move){
        this.authorization = authorization;
        this.move = move;
    }
    getAuthorization(){return this.authorization;}
    getMove(){return this.move;}
    setAuthorization(authorization){this.authorization = authorization;}
    setMove(move){this.move = move;}
}
export default MoveMessage;